test/functional/wallet_importmulti.py

```diff
@@ -119,10 +119,12 @@ def run_test(self):
         # ScriptPubKey + internal + label
         self.log.info("Should not allow a label to be specified when internal is true")
         key = get_key(self.nodes[0])
+        label = "Unsuccessful labelling for internal addresses"
```

Comment:
I think this one is touching the same lines as 3dcf1481cefae6ff2d585181578004045c3e3a1c, so I would combine these commits as well.

Reply:
@fjahr I squashed the 4 commits adding RPC getaddressinfo "labels" test coverage and improving the existing "label" tests down to a single commit at e123923e630c051f676501aab9454ecbf3bd3956. Can you please re-ACK?